counter.py

```diff
@@ -2,7 +2,8 @@
 
 from os.path import exists
 import sys
-from pickle import dump, load
+import pickle
```

Comment:
Stylistic: importing a whole library can slow down programs unnecessarily, so importing only what you need might be better